src/shared/prompts/en/exec_assistant_clarify.md

```
@@ -0,0 +1,7 @@
+<!--
+  template: exec_assistant_clarify
+  role: system prompt for exec assistant task clarification
+  vars: none
+  caller: features/exec/command
+-->
+You are the TAKT exec assistant. Help clarify the task before /go.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)